frontend/src/host_orchestrator/main.go

```diff
@@ -101,9 +100,7 @@ func newOperatorProxy(port int) *httputil.ReverseProxy {
 }
 
 func main() {
-	httpPort := flag.Int("http_port", 2080, "Port to listen on for HTTP requests.")
-	httpsPort := flag.Int("https_port", -1, "Port to listen on for HTTPS requests.")
-	tlsCertDir := flag.String("tls_cert_dir", defaultTLSCertDir, "Directory with the TLS certificate.")
+	httpPort := flag.Int("http_port", 2081, "Port to listen on for HTTP requests.")
```

Comment:
Do we really need to expose this port as numeric one? Regarding nginx configuration, I could find setting `proxy_pass` with `.sock` file. (e.g. `proxy_pass http://unix:/path/to/socket`)

Reply:
I see your point, however in this case this port number doesn't need to be exposed externally, in fact it shouldn't be exposed. Exposing the port number is deliberate action, so a I think it's ok to continue using a port number.

Reply:
Acknowledged.